cypress/e2e/SpamCheckAPI/home.cy.js

```diff
@@ -0,0 +1,58 @@
+import { SPAM_CHECK_API_BASE_URL } from '../../resources/data';
+
+describe('Spam Check API Home page', () => {
+    before(() => {
+      cy.visit(SPAM_CHECK_API_BASE_URL);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });   
+    
+    it('2. Test to check "Spam Check API" title is present', () => {
+        cy.componentVisiblityCheck('.h2', 'Spam Check API');
+    });
+ 
+    it('3. Test to check "Subscribe for free" button is present', () => {
+        cy.componentVisiblityCheck('#subscribeButton', 'Subscribe for Free');
+    });
+
+    it('4. Test to check "Live Demo" button is present', () => {
+        cy.componentVisiblityCheck('.show-code', 'Live Demo');
+    });
+
+    it('5. Test to check "code example"  is present', () => {
+        cy.componentVisiblityCheck('.code-response');
+    });
+
+    it('6. Test to click "check the documentation" hyperlink is working', () => {
+        cy.get('p > a').click()
```

Comment:
Can you please add  a class name specific to that link?

Reply:
if there is not class name get the specific text it contains to perform the click @sinchanagoudar

Reply:
it does not has the specific class so i am getting the specific text it contains to perform the click